demo/long_video_demo.py

```diff
@@ -70,6 +71,9 @@ def show_results():
         ind += 1
         prog_bar.update()
         ret, frame = cap.read()
+        if frame is None and ind == num_frames:
```

Comment:
why not just drop it?

Reply:
you know, things in videos are always inexact

Reply:
Indeed